content/blog/creating-dscc-block-storage-resources-using-dscc-ansible-playbooks.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Creating DSCC block storage resources using DSCC Ansible playbooks
+date: 2023-03-30T14:02:46.043Z
+author: Anusha Y and Sijeesh Kattumunda
+authorimage: /img/Avatar1.svg
+disable: false
+---
+In my previous blog post, I provided an introduction to Ansible playbooks for DSCC and how to use them. In this post, I will show you how to create an Ansible playbook to create block storage resources 
+Use case:
```

Comment:
...blog storage resources use case:
(type "Use case" --> "use case".)

Reply:
This is supposed to be a sub heading. I have made sure the "Use case" is presented on a new line. So will leave the casing as is.